CHANGELOG.next.toml

```diff
@@ -502,3 +502,23 @@ in non-serverless environments (e.g. via `hyper`).
 references = ["smithy-rs#2035"]
 meta = { "breaking" = true, "tada" = false, "bug" = false, "target" = "server" }
 author = "LukeMathWalker"
+
+[[smithy-rs]]
+message = """
+It is once again possible to create a service client when no default features are enabled. When default features are
```

Comment:
It was still possible last release, so really this should just say that there will be a panic if there's no connector provided or defaulted.

Might not even be worth mentioning in the changelog since it should be covered by the previous changelog entries for refactoring HTTP connectors.

Reply:
I removed them.